Common/include/geometry/primal_grid/CPrimalGrid.hpp

```diff
@@ -240,7 +240,7 @@ class CPrimalGrid {
   * \brief A virtual member.
   * \return The color of the element in the partitioning.
   */
```

Comment:
Thanks for fixing these. Weird that earlier versions of the compiler did not pick up this one.

Reply:
I think it is due to warning level rather than version. It does not show up at level 0, level 1+ is still too verbose on AD builds.